napari/_vispy/_tests/test_vispy_big_images.py

```diff
@@ -32,7 +30,6 @@ def test_big_3D_image(make_napari_viewer):
         assert np.all(layer._transforms['tile2data'].scale == s)
 
 
-@slow(10)
```

Comment:
this one?

Reply:
is this just a "safety" or is it the whole point of the test?  (i.e. is this one also just indirectly testing that we didn't compute?)

Reply:
nah, that one was added in #3407 just to avoid a regression